src/components/Layout/Header/Header.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { Link, NavLink, useLocation } from "react-router-dom";
+import "./Header.scss";
+import { getLinkStyle } from "../../../utils/Utils";
+import userImage from "../../../assets/images/icons/ic_user.svg";
+import logoImg from "../../../assets/images/icons/panda-market-logo.svg";
+import { useEffect } from "react";
+
+function Header() {
+  const location = useLocation();
+  const isAddItemPage = location.pathname === "/additem";
+  const isLoginAndSignupPage =
```

Comment:
이어지는 로직상 or 라고 표현하는 것이 좋을 것 같습니다.

```suggestion
  const isLoginOrSignupPage =
```

Reply:
확실히 로그인 또는 회원가입이 명확한 표현이네요!🫡